### DISCUSSION TITLE
Beca ingles Santander

### DISCUSSION THREAD

**Discussion by Ingrzs** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#467] [Ideas]
Ya saben sigan los consejos del MC el ingles es indispensable.
El curso abarca de nivel A1-C2 , duración 12 semanas,y solo ahi cupo para 5000.

Aqui les dejo el enlace para quien guste.

https://app.becas-santander.com/es/program/becas-santander-idiomas-online-english-courses-2022-british-council?utm_source=Email_other&utm_medium=Email&utm_campaign=BC2022_BBDD_Comparte_CTA_bt

**Comment by SujeiRodriguez** [comment]
gracias :D

**Reply by Eduar2TC** [reply]
Su campo de selección de edad es un chiste. Tienes que hacer swipe un millón de veces. Estoy desde el teléfono.

**Reply by SujeiRodriguez** [reply]
no entendí O_o...

tome el examen en ingles :'v y decía que podía ponerlo en español, pensé que era de habilidad de comprensión en ingles hahaha

**Reply by SujeiRodriguez** [reply]
si tienes computadora es mejor, se ingresan los datos fácil, aparte el examen te pide que lo hagas en la computadora

**Reply by Eduar2TC** [reply]
Perdón el reply es para Ingrz

**Reply by Ingrzs** [reply]
@Eduar2TC para nada yo complete el registro sin problemas via movil. La página tuvo problemas tal ves por el registro de muchos usuarios en ese momento

**Reply by Gracesl** [reply]
super ya postule en uno..estoy a espera de resultado, para aprender inglés.

**Comment by Takzer94** [comment]
Yo tome el ciclo pasado que termina este 27 de marzo, hacen una prueba de conocimientos y los asignan a un nivel según sus aptitudes, no abarca de A1 a C1, en mi caso me asignaron a C1, tenía 8 módulos cada uno con 5 capítulos y uno de repaso, les recomiendo tomar un capitulo por día con el fin de que no lo dejen para el final. Si vale la pena
![british council](https://user-images.githubusercontent.com/42576422/159210897-85b1479c-73df-4b3e-8a6d-b04e1eb4cd33.png)

**Reply by JuanSalvador73** [reply]
¿Y este curso es por videos grabados o texto como los de Coursera? ¿O como es la dinámica a seguir  cuando lo tomas?

**Comment by JuanSalvador73** [comment]
Gracias por el aviso :D
